client/src/App.js

```diff
@@ -8,21 +8,33 @@ export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      savedList: []
+      savedList: [],
     };
   }
 
   addToSavedList = movie => {
     const savedList = this.state.savedList;
     savedList.push(movie);
-    this.setState({ savedList });
+    this.setState({savedList});
   };
 
   render() {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route
+          exact
+          path="/" // component={MovieList}
+          render={props => (
+            <MovieList {...props} addToSavedList={this.addToSavedList} />
+          )}
+        />
+        <Route
+          path="/movies/:id"
+          render={props => (
+            <Movie {...props} addToSavedList={this.addToSavedList} />
```

Comment:
I see a bit of repetition in between your two render functions in your `Route`s here. Maybe you could make a helper method for it?
```javascript
renderComponent = Component => {
  return props => (
    <Component  {...props} addToSavedList={this.addToSavedList} />
  );
};

// ...

// and usage:
<Route 
  exact
  path="/"
  render={renderComponent(MovieList)}
/>
<Route
  path="/movies/:id"
  render={renderComponent(Movie)}
/>
```